src/Microsoft.AspNet.Mvc.Razor/RazorPage.cs

```diff
@@ -449,7 +435,7 @@ public virtual void WriteTo([NotNull] TextWriter writer, object value)
                     //
                     // Do not combine following condition with check of escapeQuotes; htmlString.ToString() can be
                     // expensive when the HtmlString is created with a StringCollectionTextWriter.
```

Comment:
comment should be changed. It mentions `htmlString.ToString()` which does not exist anymore. Also change `HtmlString` to `IHtmlContent`